Release 1.1.7

sainsachiko opened the pull request:
- [x] Update Deepvariant to 1.9.0 (anaconda-free)
- [x] Generate visual report for variant calling with Deepvariant 
- [x] Compress and index VCFs
- [x] Update locations for test data
- [x] Update CHANGELOG and version number 

<!--
# sanger-tol/variantcalling pull request

Many thanks for contributing to sanger-tol/variantcalling!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.

Learn more about contributing: [CONTRIBUTING.md](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
-->

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

juliaapolonio committed:
add visual report emmission

juliaapolonio committed:
publish the emission on the outdir

juliaapolonio committed:
add visual report as a emission of the subworkflow

juliaapolonio committed:
add output description to docs

juliaapolonio committed:
fixing target origin branch w/ change in docs

juliaapolonio committed:
remove output channels of the html report

juliaapolonio committed:
create subdir for deepvariant reports; filter publish to html files

juliaapolonio committed:
emit visual_report as output

juliaapolonio committed:
add diff file for deepvariant module

juliaapolonio committed:
change tabulations to whitespace

juliaapolonio committed:
place the deepvariant output description correctly in the docs

muffato committed:
Merge pull request #109 from juliaapolonio/add_visual_report

Add visual report

sainsachiko committed:
Update deepvariant to 1.9.0

sainsachiko committed:
Merge pull request #112 from sanger-tol/hhoang_update_deepvariant

DeepVariant was upgraded to v1.9.0, with the --vcf-stats-report flag added to run command in deepvariant/rundeepvariant/main.nf to generate the visual report (auto-generated in v1.6.1).

muffato reviewed (commented):
Hi @sainsachiko ,
Can you please update the version number and the Changelog as well ?
And on the side of this PR, please open a PR with upstream nf-core to add the visual_report output in their DeepVariant module.

sainsachiko committed:
Update CHANGELOG and version number for release 1.1.7

sainsachiko committed:
Update locations for test data

sainsachiko committed:
fix prettier check for CHANGELOG.md

sainsachiko commented:
Hi @muffato, thank you for your feedback. I’ve created a PR with the updates for release 1.1.7.

Regarding nf-core/deepvariant, nf-core has created a [separate module](https://github.com/nf-core/modules/tree/master/modules/nf-core/deepvariant/vcfstatsreport) for this functionality, rather than combining it as we currently do. Therefore, we may not need to update this part to their repo.

sainsachiko committed:
Merge pull request #114 from sanger-tol/hhoang-update-for-release-1.1.7

Update for release 1.1.7

yz533cb committed:
Remove vcf_stats_report from rundeepvariant/main.nf

yz533cb committed:
Update rundeepvariant/main.nf

yz533cb committed:
Update meta.yml

yz533cb committed:
Update deepvariant_caller.nf

Add vcfstatreport module after merging the per-chunk VCFs

yz533cb committed:
Update deepvariant_caller.nf

yz533cb committed:
Update deepvariant_caller.nf

delete the emit of vcfstatreport

yz533cb committed:
Update deepvariant_caller.nf

Add vcfstatsreport emit

yz533cb committed:
Update deepvariant_caller.nf

Edit input of VCFSTATREPORT

yz533cb committed:
Update deepvariant_caller.nf

Remove vcf_stats_report emit. Add gvcf_stats_report emit

yz533cb committed:
Update deepvariant_caller.nf

Add vcf and gvcf emits

yz533cb committed:
Update deepvariant_caller - emit
Update modules.config - add vcf_stats_report and gvcf_stats_report
Add module - vcfstatsreport

sainsachiko commented:
Hi @gq1, I have signed in to Seqera with Sanger email. Please re-add me, sincerely thank you!

yz533cb committed:
Update modules.config

publishDir change to variant_calling/report

yz533cb committed:
Merge pull request #116 from yz533cb/yz533cb-patch-1

VCF stats report for whole VCF/GVCFs

yz533cb committed:
Compress and index VCF file

yz533cb committed:
Update deepvariant_caller.nf

yz533cb committed:
Update variantcalling.nf

yz533cb committed:
Update deepvariant_caller.nf

yz533cb committed:
Update deepvariant_caller.nf

sanger-tolsoft committed:
[automated] Fix linting with Prettier

yz533cb committed:
Revert "Update deepvariant_caller.nf"

This reverts commit c89bbd6dd3e0b58cbd528ad4f75dc833410a0365.

yz533cb committed:
Revert "[automated] Fix linting with Prettier"

This reverts commit 755172db719c8b1f34b05a7d9230ebf5610fbf80.

Revert the modification made by [automated fix linting with Prettier]

yz533cb committed:
Run prettier on modules/nf-core/tabix/bgzip/meta.yml

yz533cb committed:
Add diff

yz533cb committed:
Modify bgzip diff

yz533cb committed:
Remove bgzip/diff file because the module has updated upstream

yz533cb committed:
Update tabix/bgzip in modules.json

yz533cb committed:
Update the deepvariant-rundeepvariant.diff file with patch command

yz533cb committed:
update .diff file of deepvariant using patch

yz533cb committed:
Use Prettier to format modules.json file

yz533cb committed:
Run patch command again without changing modules.json file

yz533cb committed:
Merge pull request #118 from sanger-tol/yz12-compress_and_index

Compress and index vcf/gvcf files; update deepvariant-rundeepvariant.diff file

sainsachiko committed:
Update CHANGELOG.md

sainsachiko committed:
Update output.md for variant calling

sanger-tolsoft committed:
[automated] Fix linting with Prettier

sainsachiko committed:
Update software dependency in CHANGELOG.md

sainsachiko committed:
Update file name of visual report in output.md

yz533cb committed:
Merge pull request #120 from sanger-tol/hhoang-prep-release-1.1.7

Update documents for release 1.1.7

sainsachiko commented:
Hi @muffato, we have finalize the PR for this release with:
- [x] Update Deepvariant to 1.9.0 (anaconda-free)
- [x] Generate visual report for variant calling with Deepvariant 
- [x] Compress and index VCFs
- [x] Update locations for test data
- [x] Update CHANGELOG and version number 

Please kindly review this, thank you so much!

muffato committed:
Updated the author list and fixed the CFF file

muffato committed:
today's date

muffato committed:
Merge pull request #121 from sanger-tol/author_list

Updated the author list and fixed the CFF file

muffato committed:
bugfix: the information is in the ch_genome_index_fai channel

muffato committed:
Merge pull request #122 from sanger-tol/fix_csi_tbi

bugfix: the information is in the ch_genome_index_fai channel
